zone/groups.cpp

```diff
@@ -1017,6 +1017,27 @@ void Group::GetBotList(std::list<Bot*>& bot_list, bool clear_list)
 	}
 }
 
+void Group::GetRawBotList(std::list<uint32>& bot_list, bool clear_list)
```

Comment:
This should return what you need, not return by passing a value by reference.

Eg

```
std::list<uint32> Group::GetRawBotList(bool clear_list)
```

Is there no other way to reference bots in a group by memory than to query the database?

Reply:
It looks like it only uses the DB on this for "offline" or dead bots.